### ISSUE TITLE
Staging Review finding: Breadcrumbs are missing

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#127620]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BIO - Pingwind
**Product:** VA Form 21-8940 Veteran's Application for Increased Compensation Based on Unemployability
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can’t determine where a page or screen is within VA.gov or VA Mobile App.](https://design.va.gov/about/experience-standards/#user-can-determine-where-a-page-is-within-the-vagov-website)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

Almost all pages on VA.gov need a breadcrumb to allow users to navigate back in the page hierarchy. The form is missing breadcrumbs, which is also creating a formatting issue where the page title is aligned too close to the header.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="707" height="318" alt="Image" src="https://github.com/user-attachments/assets/4e432c65-f027-42f2-806f-f3b8aca4fc37" />

</details>

## Recommended action

Since you are using the minimal form header, use the back link where the breadcrumbs would go.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.8 Location - Level AAA](https://www.w3.org/TR/WCAG22/#location)
- **Modality:** N/A
- **Design System Component:** [Breadcrumbs component](https://design.va.gov/components/breadcrumbs)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
